test/e2e/framework/ginkgowrapper/wrapper.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I finally pulled this pkg into our repo for two reasons:
1) It's the only pkg we use from kubernetes, so we avoid downloading the whole kubernetes project just for 1 file
2) It created an issue where ginkgo was compiled and initialized twice (once for the one in our tree, and another one for the "vendored" version in kubernetes. Making the test binary fail hard at start.